docs/faq/faq-felizandfable.md

```diff
@@ -0,0 +1,73 @@
+In a nutshell Fable.React and Feliz are two F# libraries which perform a similar function:
+
+* Provide a typesafe F# wrapper to allow you to interact with React.
+* Provide a way to create your own wrappers around existing React components.
+* Provide a DSL for you to consume and create React wrappers in a consistent way.
+
+## DSL differences
+The main distinction between the two libraries is that Fable.React follows a layout as follows:
+
+```fsharp
+element [ prop; prop ] [ childElement; childElement ]
+```
+
+For example:
+
+```fsharp
+h1 [ Style [ Color "Tomato" ] ] [
+    p [] [ text [ Value "Hello" ] [] ] // no props
+    p [] [ text [ Value "Another paragraph" ] [] ] // no props
+    h2 [ Style [ Color "Blue" ] ] [] // no child elements
+]
+```
+
+In this example, `h1` is the "top level" element, with a single attribute and three children, two of which have their own children.
+
+Feliz adopts a different style, in which instead of an element having two lists, there is only one. The list directly contains *either* attributes *or* child elements, but not both:
+
+The above snippet would convert into Feliz as follows:
+
+```fsharp
+Html.h1 [
+    prop.style [ style.color "Tomato" ]
+    prop.children [
+        Html.p [ prop.text "Hello" ]
+        Html.p [ prop.text "Another paragraph" ]
+        Html.h2 [ prop.style [ style.backgroundColor "Blue" ] ]
+    ]
+]
+```
+
+The `prop.children` function is required when mixing and matching attributes and elements:
+
+```fsharp
+Html.h1 [ // this is fine - just props underneath h1
+    prop.style [ style.color "Tomato" ]
+    prop.title "foo"
+]
+
+Html.h1 [ // fine - just elements underneath h1
+    Html.p [ prop.text "Hello" ]
+]
+
+Html.h1 [ // not fine - can't mix and match attributes and elements
+    prop.style [ style.color "Tomato" ]
+    Html.p [ prop.text "Hello" ]
+]
+```
+
+In order to allow both attributes and elements in the same list, Feliz introduces the `children` node:
```

Comment:
It feels like you are repeating the same sentence has above the snippets.

You can perhaps remove that sentence and include the everything in a single snippet